### PR TITLE
new icon: rollup (original, original-wordmark, plain, plain-wordmark, line, line-wordmark)

### PR DESCRIPTION
**Pull request by PantheraRed**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://rollupjs.org/guide/en/

### CODE REVIEW

**Comment by lunatic-fox** [comment]
Hello there @PantheraRed!
I have a suggestion about `plain` icon version:
<div align="center">
  rollup-plain-suggestion
</div>

What do you think?
> \* The icon is colored for preview only, but we have to exclude any fill in the final product.

**Comment by lunatic-fox** [comment]
What do you think about this suggestion @Snailedlt, @Panquesito7?
> Feel free to mark more maintainers.

**Comment by Snailedlt** [comment]
@lunatic-fox Love it!
:ship: :it:

**Comment by Snailedlt** [comment]
Any of you wanna try and make a line icon too?

"How do I do that?" I hear you ask.
This is how: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294 :)

**Comment by lunatic-fox** [comment]
🙂👍🏼
<div align="center">
  rollup-plain-suggestion
</div>

**Comment by Snailedlt** [comment]
@PantheraRed can you add those two icons to the PR and let me know when it's there please? :)

**Comment by PantheraRed** [comment]
Done @Snailedlt

**Comment by Snailedlt** [comment]
@PantheraRed just found the wordmark logo too on their website. You wanna add original, plain and line verdions of that too?

I'm on the phone so can't share a previw or direct link to it, but the svg is top left corner on this page: https://rollupjs.org/guide/en/

**Comment by PantheraRed** [comment]
I am sure not sure if they have an actual wordmark logo. It's not in their repository. But if want me to add them sure. @Snailedlt 

Screenshot from website:
Screenshot_20221104-043406

**Comment by Snailedlt** [comment]
Isn't the one you screenshot there their wordmark logo?

**Comment by PantheraRed** [comment]
> Isn't the one you screenshot there their wordmark logo?

I don't think so. It defers from platform to platform. It's not on their repository's readme either. @Snailedlt 

Windows:
windows

Android:
android

**Comment by PantheraRed** [comment]
Sure if you're able to @Snailedlt.

File: devicon.json
Comment (Snailedlt) [inline devicon.json:6647]:
```suggestion
            "rollupjs",
            "rollup.js"
```

File: devicon.json
Comment (Snailedlt) [inline devicon.json:6651]:
```suggestion
            "build",
            "javascript",
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looking good for the most part. Added some suggestions for devicon.json, the icons themselves look perfect though 💯 

Sidenote: I reached out to rollup devs about the wordmark icons in this issue: https://github.com/rollup/rollupjs.org/issues/171 . Hopefully we can get a response from them soon :)

**Comment by PantheraRed** [comment]
@Snailedlt any idea what happened here?

**Review by Snailedlt — DISMISSED** [review]
Alright @PantheraRed I think this looks good now! ✔️ 
We still need to wait for another review before merging. But if that comes quickly... do you prefer to merge, or wait a few days and see if we get an answer regarding the wordmark version?

**Comment by PantheraRed** [comment]
Yes, I'd like to wait for few days and see if we get any response regarding the rollup-wordmark. Thank you. :+1: @Snailedlt

**Comment by Snailedlt** [comment]
@PantheraRed The main maintainer of rollup said he prefferred the windows version, so I think we should go ahead with that one as original-wordmark then :)

Ref: https://github.com/rollup/rollupjs.org/issues/171#issuecomment-1306103030

If it's not too much trouble, please make wordmark versions of line and plain icons based on the windows version you showed earlier :)

As mentioned earlier, you can follow this guide to mkae line icons: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294


If you need any help, just ask :)

**Comment by PantheraRed** [comment]
@Snailedlt
If I'm not mistaken according to this https://github.com/rollup/rollupjs.org/issues/171#issuecomment-1306369630 you wanted to go with the android version. Are you sure you want me to go with Windows version?

**Comment by Snailedlt** [comment]
@PantheraRed ofc, I'm just testing you ;)
<details>
Jk, I guess I was in a hurry when writing this... Yes I meant the Android version :)
Lucky for us that you noticed it
</details>

**Comment by PantheraRed** [comment]
Thanks to @lunatic-fox for plain and line logos. :+1:

File: icons/rollup/rollup-line-wordmark.svg
Comment (Snailedlt) [inline icons/rollup/rollup-line-wordmark.svg:1]: Can we make the lines a bit thicker for the logo?

I also wonder if it would look better with hollow text, so the whole icon is lines. What do you think?
This is not a must though. In fact most line-wordmark icons don't have hollow/line text.
I will accept it even if the text is unchanged :)

If you wanna go for hollow text, you can use the djangorest PR as an example: https://github.com/devicons/devicon/pull/1407/files#diff-75a0ca900d5867593762acd10b934d39e826fe5c00a097f0e93a2f9a6380b53a

Examples of hollowed text:
Djangorest: https://github.com/devicons/devicon/pull/1407/files#diff-75a0ca900d5867593762acd10b934d39e826fe5c00a097f0e93a2f9a6380b53a
Unity: https://github.com/devicons/devicon/blob/develop/icons/unity/unity-line-wordmark.svg

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Neat!
Just a couple small suggestion for the line-wordmark version

File: icons/rollup/rollup-line-wordmark.svg
Comment (PantheraRed) [reply icons/rollup/rollup-line-wordmark.svg:1]: Sure, I'll create a sample. :+1:

Comment (PantheraRed) [reply icons/rollup/rollup-line-wordmark.svg:1]: How's this, @Snailedlt?
test

Comment (PantheraRed) [reply icons/rollup/rollup-line-wordmark.svg]: @Snailedlt done

**Review by Snailedlt — COMMENTED** [review]
LGTM! ✔️ 
Thank you for a great contribution! 🚀 
And thank you @lunatic-fox for the plain and line versions

**Review by Snailedlt — DISMISSED** [review]

File: icons/rollup/rollup-line.svg
Comment (lunatic-fox) [inline icons/rollup/rollup-line.svg:1]: Remove `style` and `xml:space`. 🙂
This suggestion does not change the appearance of the icon, here's the preview:
<div align="center">
  
</div>

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M22 2c-1.3 0-2.4 1.1-2.4 2.4v99.2c-1.3 3.4-2.1 6.2-2.6 8.5v.1c-.6 2.7-.7 4.8-.5 6.3 0 .3.1.6.1.9.1.8.4 1.5.8 2.1.2.4.4.7.6 1 .3.4.6.7.8 1l.7.7c.1 0 .1.1.1.1.2.1.4.3.6.4l.1.1h.1c2.8 1.7 6.6 1.5 9.3.2h79.4c1.8 0 2.9-1.9 2.1-3.5L90.4 80.4c-.3-.7-.1-1.5.5-1.8 12.3-7.1 20.6-20.3 20.6-35.6 0-7.3-1.9-14.2-5.3-20.1-.3-.5-.6-1-.8-1.4v-.1C98.3 9.9 85.6 2.2 71.2 2H22zm0 1h49.2c11.9.2 22.5 5.6 29.7 14l-.1-.1-.1-.1c-.3-.2-.5-.4-.8-.6-.1-.1-.2-.2-.4-.3-.1-.1-.2-.1-.2-.2-.1-.1-.3-.2-.4-.3-.1 0-.1-.1-.2-.1-.1-.1-.3-.2-.4-.3-.3-.2-.7-.4-1-.6-.1 0-.1-.1-.2-.1-.4-.2-.7-.4-1.1-.6-.1 0-.2-.1-.3-.1-.1-.1-.2-.1-.4-.2s-.3-.2-.5-.2c-.1 0-.2-.1-.3-.1-.1 0-.1-.1-.2-.1-.4-.1-.7-.3-1.1-.4h-.1c-.1 0-.2-.1-.3-.1h-.1c-.3-.1-.7-.2-1-.3-.1 0-.2-.1-.4-.1-.1 0-.2-.1-.4-.1-.2-.1-.4-.1-.6-.2-.1 0-.1 0-.2-.1-.1 0-.2 0-.2-.1-.4-.1-.8-.2-1.2-.2h-.4c-.5-.1-.9-.1-1.4-.2h-.4c-.2 0-.4 0-.6-.1h-.4c-.1 0-.1 0 0 0H84c-.9 0-1.7 0-2.5.1h-.3c-1.3.1-2.5.2-3.6.5-.8.2-1.6.4-2.3.7-1.5.6-3 1.4-4.5 2.5-.7.6-1.5 1.2-2.2 1.9l-.3.3c-.3.3-.7.7-1 1.1-.7.8-1.4 1.6-2.2 2.5-.1.1-.2.3-.3.4-.6.8-1.2 1.6-1.9 2.5-.3.4-.6.9-1 1.4-.5.7-1 1.5-1.5 2.2-.3.5-.6 1-1 1.5-.7 1.1-1.4 2.2-2.1 3.4-3.5 5.7-7.8 13.2-12.9 22.4-2.6 4.7-5.4 9.7-8.5 15.5 0 .1-.1.2-.1.3-1.5 2.7-2.8 5.2-4.1 7.6-5.1 9.7-8.5 17.2-10.8 23V4.4c-.3-.8.3-1.4 1.1-1.4zm61.9 9.2c.8 0 1.5 0 2.3.1h.3c5.6.6 11 3 15.3 6.9-3.1-1.9-6.7-3.3-10.3-3.9l-1.8-.3h-.2c-.5-.1-1.1-.1-1.6-.1h-2.4c-.9.1-1.7.2-2.6.3-.7.1-1.3.3-2 .5-3.8 1.1-6.9 3.4-8.4 7.1-2.5 5.9-.4 12.2 3.1 16.8 1.7 2.3 3.8 4.2 5.8 5.5.2.1.3.2.5.3h.1c.1.1.2.1.3.1.3.2.6.3 1 .5.3.1.7.3 1 .4.1 0 .2.1.3.1l1.2.3h.8c.2 0 .4 0 .6-.1.3 0 .5-.1.7-.2l.6-.3c.4-.2.7-.5.9-.8.5-.6.8-1.4 1-2.3.3-1.7 0-3.6-.5-5.6.3.7.9 1.7 1.1 2.2.7 1.8.9 3.2.7 4.3v.2c-.1.4-.2.8-.4 1.2-.8 1.6-2.6 3.2-5.6 5.7-2.9 2.4-6.9 5.7-12 10.6-10.2 9.8-20.4 24.9-28.5 37.8-4 6.4-7.6 12.3-10.2 16.7-1.3 2.2-2.5 4-3.4 5.3-.4.7-.8 1.2-1.1 1.6-.3.4-.6.6-.6.6-2.4 1.4-6.3 1.7-8.9.1l-.1-.1c-.2-.1-.3-.2-.5-.4l-.1-.1-.6-.6c-.3-.3-.6-.7-.9-1.1-.5-.7-.8-1.5-1-2.3-.1-.3-.1-.7-.2-1.1-.2-1.4-.1-3.4.5-6.1.6-2.7 1.7-6.2 3.4-10.7.5-1.3 1-2.6 1.7-4.1 2-4.6 4.6-10.1 7.9-16.5 1.6-3.2 3.4-6.5 5.4-10.2 0-.1.1-.1.1-.2 3.2-5.8 5.9-10.9 8.5-15.6 1.9-3.4 3.7-6.7 5.3-9.6 3.8-6.7 7-12 9.7-16.3l.6-.9c.7-1 1.3-2 1.9-2.9l.6-.9c.8-1.1 1.5-2.1 2.3-3 .7-.9 1.4-1.6 2-2.3.5-.5.9-1 1.4-1.4.7-.7 1.4-1.3 2.2-1.8 1.4-1.1 2.8-1.8 4.3-2.3.6-.2 1.1-.4 1.7-.5.3-.1.7-.2 1.1-.2.3-.1.6-.1.9-.2.6-.1 1.2-.1 1.8-.2.9.1 1.9 0 3 0zm3.9 3.7c6.4.3 13.3 3 17.6 7.4 3.3 5.8 5.1 12.5 5.1 19.6 0 14.9-8.1 27.8-20.1 34.7-1.1.6-1.5 2-.9 3.1l20.8 41.1c.5.9-.2 2-1.2 2H31l.2-.2c.3-.4.7-1 1.2-1.6.9-1.3 2-3.1 3.3-5.2 0-.1.1-.1.1-.2 2.7-4.4 6.2-10.3 10.2-16.7 8-12.6 18.3-27.7 28.3-37.3 5.1-4.8 9-8.1 11.9-10.5 2.9-2.5 4.8-4.1 5.8-6 .2-.5.4-.9.5-1.4.3-1.5.1-3.1-.7-5.1-1-2.7-3-6.3-5.8-11.5-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c0 0-3.4 4.8-8.3 11.9-.2-.2-.4-.4-.6-.7-1.8-2.4-3.2-5.2-3.7-8.1 0-.3-.1-.5-.1-.8-.3-2.3-.1-4.6.9-6.9 2-4.7 6.7-7 12.1-7.3.7-.2 1.5-.2 2.3-.2zm-2.3 13.2c.5.9 2 4 3 7.7.6 2.1 1 4.3.8 6-.1.8-.3 1.5-.7 2-.4.5-.9.8-1.7 1-.3.1-.6.1-1 0-.3 0-.7-.1-1.1-.2-.1 0-.1 0-.2-.1-.3-.1-.7-.2-1.1-.4-.1 0-.2-.1-.3-.1-.3-.2-.7-.3-1-.5-.2-.1-.5-.3-.7-.5-.1-.1-.3-.2-.4-.3-1.2-.9-2.4-1.9-3.5-3.2 4.6-6.7 7.6-10.9 7.9-11.4zM18 112.2z" fill="#f33"/></svg>

```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Optimize:
- `plain-wordmark`
- `line`

**Review by Snailedlt — APPROVED** [review]

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for the great contribution! 🎉

**Pull request merged**
